scripts/dist.js

```diff
@@ -1,49 +1,52 @@
 const fsp = require("fs").promises;
 const path = require("path");
 
-async function main() {
+async function main () {
     // Get lists of Brahmic and roman schemes
     // Each of bschemes and rschemes is an array of which each element is of
     // the form [filename, filepath]
     const [bschemes, rschemes] = await Promise.all(
-        ["brahmic", "roman"].map(async x => {
-            const dirpath = path.join(__dirname, "src", "schemes", x)
-            let paths = [];
+        ["brahmic", "roman"].map(async (x) => {
+            const dirpath = path.join(__dirname, "..", "src", "schemes", x);
+            const paths = [];
             for (const filename of await fsp.readdir(dirpath)) {
                 paths.push([filename, path.join(dirpath, filename)]);
             }
             return paths;
-        })
+        }),
     );
 
     // Get arrays of Brahmic and roman scheme file contents
     // Each of bfiles and rfiles is an array of which each element is of the
     // form [schemeName, fileContents]
-    const [bfiles, rfiles] = await Promise.all([bschemes, rschemes].map(x =>
+    const [bfiles, rfiles] = await Promise.all([bschemes, rschemes].map((x) =>
         Promise.all(x.map(async ([filename, filepath]) => {
-            let schemeName = filename.split(".");
+            const schemeName = filename.split(".");
```

Comment:
I guess this is more idiomatic in JavaScript. I used `let` because I was thinking of immutability in other languages like Rust, but that is clearly not the case in JS (where `const` only prevents reassignment).

Reply:
I totally agree — it trips me up sometimes too.